src/components/movies/moviesNav/genreDropdown/fantasy/fantasy.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function FantasyGenre(props: any) {
```

Comment:
Also das mit den Genres musss du in eine Func stecken. Jeweils eine Component erscheint mir unnötig.
```
const FantasyGenre = ({ onFilter, title } : { onFilter: Function, title: string}) => (
<button
      className="dropdown-item btn-sm"
      type="button"
      id="Fantasy"
      onClick={onFilter}
    >
      {title}
    </button>
)
```